### ISSUE TITLE
My Catalog "Visit Admin" and "Visit Book" Link don't appear on Keyboard Focus

### ISSUE THREAD

**Issue by avaaec** [pressbooks/pressbooks#1781]
### Steps to Reproduce

1. Navigate to My Catalog page with an account that has at least one book
2. Navigate to the table in the middle of the page
3. Use the tab key to navigate through the focusable elements of the table
4. Notice that your focus lands on the "Visit Admin" and "Visit Book" links under the title of each book but the links do no appear visually
5. Use the mouse to hover over that cell and notice how they appear during a mouse over.

**Expected behavior:** 
Visually hidden elements should be visible when focused. 
**Actual behavior:** 
The links do not appear visually when focused with the keyboard. This causes a lot of problems for visual keyboard users who do not have the aid of a screen reader or other assistive technology to help them see the element their focus is on. When they focus these links they have no way of knowing where their focus is or what the focused element does, because they cannot see the text. This is confusing and kind of a bummer.

**Comment by mcgratay** [comment]
Tested in Firefox, Chrome, and Safari - These links still don't appear on keyboard focus.

**Comment by SteelWagstaff** [comment]
Sorry -- mistakenly tagged this issue in my PR. I did not address this one in my fixes! I mean

**Comment by SteelWagstaff** [comment]
What I did fix was moving the 'edit profile' and 'visit catalog' links out of the H1 at the top of the catalog page. Those links do now appear when a user tabs through the page, but that's not what this ticket is about. Sorry!

**Comment by connerbw** [comment]
Fix adds `has-row-actions` to column header classes

![confusing and kind of a bummer](https://user-images.githubusercontent.com/812192/67704289-2fd88480-f98b-11e9-80c0-8472f28007cf.png)

**Comment by avaaec** [comment]
This is a UW ATS priority 1 issue

**Comment by connermwallace** [comment]
University of Washington Accessibility Technology Services has inspected this issue and we have verified that it has been fixed. Version: 5.3.2.
